src/SDKs/Network/Network.Tests/Tests/CheckConnectivityTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Management.Network.Models;
+using Microsoft.Azure.Management.Resources;
```

Comment:
@irrogozh please use the latest Resource manager nuget package >=1.6.0 to record your tests